genesis/model/resolver/handler.go

```diff
@@ -22,23 +22,28 @@ import (
 	"github.com/insolar/insolar/genesis/model/object"
 )
 
-type resolverHandler struct {
+// Handler can resolver references from any allowed scopes
+type Handler struct {
 	globalResolver  *globalResolver
 	childResolver   *childResolver
 	contextResolver *contextResolver
 }
 
-// NewResolverHandler creates new resolverHandler instance.
-func NewResolverHandler(p object.Parent) Resolver {
-	return &resolverHandler{
+// NewHandler creates new resolverHandler instance.
+func NewHandler(p object.Parent) *Handler {
+	return &Handler{
 		globalResolver:  GlobalResolver,
 		childResolver:   newChildResolver(p),
 		contextResolver: newContextResolver(p),
 	}
 }
 
 // GetObject resolve object by its reference and return its proxy.
-func (r *resolverHandler) GetObject(ref *object.Reference, classID string) (object.Proxy, error) {
+func (r *Handler) GetObject(reference interface{}, classID interface{}) (interface{}, error) {
```

Comment:
Add description to doc: why we use interface{} types for reference, class and proxy